rllib/algorithms/dt/tests/test_segmentation_buffer.py

```diff
@@ -0,0 +1,367 @@
+import unittest
```

Comment:
**TODO**: There is no unittest that actually tests the zero padding which is a very important piece of the SegmentationBuffer logic. Please add a unittest for it.

Reply:
added